### PR TITLE
Urban rural disaggregation

### PR DESCRIPTION
**Pull request by a-w-beck**
Functions to map flows with FIPS code Location values to Census-data-derived urban population percentages, and then split flows into 'urban' and 'rural' components plus append a secondary Compartment label.

Also migrated a collection of Location- and FIPS-related functions from common.py to location.py.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -13,8 +13,8 @@
     include_package_data=True,
     install_requires=[
         'fedelemflowlist @ git+https://github.com/USEPA/Federal-LCA-Commons-Elementary-Flow-List.git@develop#egg=fedelemflowlist',
-        'esupy @ git+https://github.com/USEPA/esupy.git@develop#egg=esupy',
-        'StEWI @ git+https://github.com/USEPA/standardizedinventories.git@develop#egg=StEWI',
+        'esupy @ git+https://github.com/USEPA/esupy.git@urban_rural_classify#egg=esupy',
+        'StEWI @ git+https://github.com/USEPA/standardizedinventories.git@urban_rural_classify#egg=StEWI',
```
Comment (catherinebirney) [inline setup.py]: if you have editable versions of packages installed locally, is it necessary to change these branches to urban_rural_classify?

Comment (catherinebirney) [reply setup.py]: I see @bl-young changed it back

**Review by catherinebirney — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (bl-young) [reply setup.py]: yep - this isn't necessary anymore as there's nothing on that esupy branch needed in flowsa

**Review by bl-young — APPROVED** [review]
Confirmed I could create an FBS using EPA_NEI_Onroad data with `apply_urban_rural` set to True:

```
2022-01-26 16:11:40 INFO     Retrieving Flow-By-Activity for datasource EPA_NEI_Onroad in year 2017
2022-01-26 16:11:53 INFO     Splitting EPA_NEI_Onroad into urban and rural quantities by FIPS.
2022-01-26 16:12:57 INFO     Cleaning up EPA_NEI_Onroad FlowByActivity
2022-01-26 16:13:23 INFO     Preparing to handle activity_set_1 in EPA_NEI_Onroad
2022-01-26 16:14:28 INFO     Searching for and allocating FlowAmounts for any parent NAICS that were dropped in the subset to NAICS_6 child NAICS
...
2022-01-26 16:14:28 INFO     Comparing Flow-By-Activity subset by activity and geography to the subset Flow-By-Sector FlowAmount total.
2022-01-26 16:14:28 INFO     Comparing FBA activity_set_1 national subset to FBS results. Details in Validation Log
2022-01-26 16:14:28 INFO     FlowBySector FlowAmount for EPA_NEI_Onroad activity_set_1 emission/air/troposphere/rural at 00000 is equal to the FlowByActivity FlowAmount
2022-01-26 16:14:28 INFO     FlowBySector FlowAmount for EPA_NEI_Onroad activity_set_1 emission/air/troposphere/urban at 00000 is equal to the FlowByActivity FlowAmount
```

**Pull request merged**